desc/equilibrium/equilibrium.py

```diff
@@ -1221,6 +1221,50 @@ def map_coordinates(
             **kwargs,
         )
 
+    def get_rtz_grid(
```

Comment:
if this only needs to be here to avoid circular import can we make it private at least?